src/styles/globals.css

```diff
@@ -89,6 +89,7 @@ input[type='time'],
 input[type='url'],
 input[type='week'],
 select:focus,
+input,
```

Comment:
Not sure if there are any downsides to targeting all inputs, or if we instead should add `type="text"`, or even `type="search"` (checking all browsers to see if any of them adds any weirdness).

Reply:
I think its more on specificity issues tbh

Reply:
Okey, i´ll investigate👍